libs/components/help-inline/src/lib/modules/help-inline/help-inline.modern.component.scss

```diff
@@ -37,7 +38,10 @@
     --sky-override-help-inline-border-radius,
     var(--sky-border-radius-s)
   );
-  color: var(--sky-color-icon-action);
+  color: var(
+    --sky-override-help-inline-icon-color,
```

Comment:
Is this override needed? Does `--sky-color-background-icon_matte-action-heavy` point to the same color as `--sky-color-icon-action`? If so - we can just switch them out.

Reply:
I'd split them up jic there were any changes down the line for whatever reason for icon colors. But I'm going to ask design about the likelihood of that happening.

Reply:
Historically - we have not worried about that as it would cause a large number of unneeded overrides - and any change in the tokens would be caught in a design token update and we could add the override then.